pkg/apis/hive/v1/hiveconfig_types.go

```diff
@@ -143,7 +143,6 @@ type ManageDNSAWSConfig struct {
 	// AWS Route53. It will need permission to manage entries for the domain
 	// listed in the parent ManageDNSConfig object.
 	// Secret should have AWS keys named 'aws_access_key_id' and 'aws_secret_access_key'.
-	// +optional
 	CredentialsSecretRef corev1.LocalObjectReference `json:"credentialsSecretRef,omitempty"`
```

Comment:
Since these are not optional, the `omitempty` should be removed as well.

Reply:
Fixed